packages/dashboard/src/components/Wizard/examples/SetupWizard.Util.ts

```diff
@@ -0,0 +1,222 @@
+import { ISubwayNavStep, SubwayNavStepState } from '../../SubwayNav/SubwayNav.types';
+import { IWizardStepProps } from '../Wizard.types';
+
+export interface ISetupWizardState {
+  currStep: IWizardStepProps;
+  currSubStep: IWizardStepProps | undefined;
+  steps: IWizardStepProps[];
+}
+
+/**
+ * generate Random id
+ */
+export function generateRandomId(): string {
+  return (
+    Math.random()
+      .toString(36)
+      .substring(2) + new Date().getTime().toString(36)
+  );
+}
+
+// Go to a given parent step -> substep
+export function goToStep(
+  currentState: ISetupWizardState,
```

Comment:
so this utility is specifically for setup wizard? Seems pretty generic to all wizards.

Reply:
Correct, I started to move generic methods to util.  
But anyway, since this is just example code, it is ok for now.  

When I work on making the control unmanaged, I can consider making it generic.